lib/editing/editor.dart

```diff
@@ -126,7 +126,18 @@ class Position {
 }
 
 abstract class CodeCompleter {
-  Future<List<Completion>> complete(Editor editor);
+  Future<Completions> complete(Editor editor);
+}
+
+class Completions {
```

Comment:
Hm.. kind of agree, but `Completion` is kind of the dartpad version of `HintResult` from codemirror.dart

So therefore I thought it would be logical to call the dartpad version of `HintResults` then `Completions`